tests/functional/c/consider/consider_using_f_string.py

```
@@ -0,0 +1,107 @@
+def return_list():
+    return PARAM_LIST
+
+
+def return_dict():
+    return PARAM_DICT
+
+
+def print_good():
+    print("String {}, {} or {}".format(*PARAM_LIST))
+    print("String {}, {}, {} or {}".format(*PARAM_LIST_SINGLE, *PARAM_LIST))
+    print("String {Param}, {}, {} or {}".format(Param=PARAM_1, *PARAM_LIST))
```

Ruff (0.11.9)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)